src/Diagnostics.RuntimeHost/Controllers/DevOpsController.cs

```diff
@@ -88,10 +88,10 @@ public async Task<IActionResult> GetResourceProviderConfig(string resourceProvid
             ResourceProviderRepoConfig resourceProviderRepoConfig = await _devOpsClient.GetRepoConfigsAsync(resourceProviderType);
             if (resourceProviderRepoConfig == null)
             {
-                var noConfigResponse = new { GraduationEnabled = false, AutoMerge = false };
+                var noConfigResponse = new { GraduationEnabled = false, AutoMerge = false, Organization = "", Repository = "", Project = "" };
                 return Ok(noConfigResponse);
             }
-            var response = new { GraduationEnabled = true, AutoMerge = resourceProviderRepoConfig.AutoMerge };
+            var response = new { GraduationEnabled = true, AutoMerge = resourceProviderRepoConfig.AutoMerge, Organization = resourceProviderRepoConfig.Organization, Repository = resourceProviderRepoConfig.Repository, Project = resourceProviderRepoConfig.Project };
```

Comment:
I believe you have to send folderpath as well. Sometimes teams may have detectors/gists stored under a subfolder.